pandas/tseries/offsets.py

```diff
@@ -687,11 +733,15 @@ def rollforward(self, dt):
                 return self._prev_opening_time(dt)
         return dt
 
+    def _get_closing_time(self, dt):
+        for i, st in enumerate(self.start):
+            if st.hour == dt.hour and st.minute == dt.minute:
+                return dt + timedelta(
+                    seconds=self._get_business_hours_by_sec(st, self.end[i]))
+        raise ValueError("dt should be a starting time")
```

Comment:
Can you add a test where this is hit?

Reply:
You mean where `ValueError` is raised here? `ValueError` would never be raised here if only public methods are called.

Reply:
Well can this `ValueError` get floated up to the user? If so, we should have a test to catch it.

Reply:
Only if user calls `_get_closing_time` him/herself

Reply:
@mroeschke so should this be tested? If yes, do we just run `_get_closing_time` directly in the test function?

Reply:
So if this error can never be hit by the user using public methods, I'm still not sure if we entirely need this error.

Reply:
Should I remove the raise statement? Or add a comment to indicate that it would never be hit?

Reply:
If `if st.hour == dt.hour and st.minute == dt.minute:` is always guaranteed to get hit, then you can remove the error.

Reply:
I have removed the error and added a comment.